services-toolkit/tutorials/working-with-the-bitnami-services.hbs.md

```diff
@@ -0,0 +1,100 @@
+
+## In a nutshell
+
+Let's begin with a bird's-eye overview of the pieces we'll use during the course of this tutorial and how they all fit together. 
+
+![Diagram shows a high-level overview of the out of the box bitnami services](../../images/stk-dynamic-provisioning-bitnami-services.png)
+
+There are a couple of key things to note in the diagram.
+
+1. There are only a couple of pieces requiring user input - the creation of the `ClassClaim` and the creation of the `Workload` (which is configured to refer to the `ClassClaim`)
+1. The lifecycle of `ClassClaim` and the lifecycle of `Workload` are separate, allowing updates to be made to one without affecting the other
+1. What happens behind the Dynamic Provisioning box is opaque - this is intentional as Application Operators and Developers shouldn't need to concern themselves with the inner workings and configurations of service instances
+
+## Tell me more about these services
+
+TODO: flesh this out
+
+* immediately available
+  * Great for getting started with services on TAP
+* unmanaged
+  * run on the cluster
+* powered by dynamic provisioning of bitnami helm charts
+* may not suit every use case
+
+## Discovery
+
+```console
+$ tanzu service class list
+
+  NAME                  DESCRIPTION                                                                     
+  mysql-unmanaged       MySQL by Bitnami                                                                
+  postgresql-unmanaged  PostgreSQL by Bitnami                                                           
+  rabbitmq-unmanaged    RabbitMQ by Bitnami                                                             
+  redis-unmanaged       Redis by Bitnami
+```
+
+```console
+$ tanzu service class get postgresql-unmanaged
+
+NAME:           postgresql-unmanaged
+DESCRIPTION:    PostgreSQL by Bitnami
+READY:          true
+
+PARAMETERS:
+  KEY        DESCRIPTION                                                  TYPE     DEFAULT  REQUIRED  
+  storageGB  The desired storage capacity of the database, in Gigabytes.  integer  1        false
+```
+
+## Claiming
+
+```console
+$ tanzu service class-claim create psql-1 --class postgresql-unmanaged --parameter storageGB=3
+
+Creating claim 'psql-1' in namespace 'dev-team-1'.
+Please run `tanzu services class-claims get psql-1 --namespace dev-team-1` to see the progress of create.
+```
+
+```console
+$ tanzu service class-claim get psql-1
+
+Name: psql-1
+Namespace: dev-team-1
+Claim Reference: services.apps.tanzu.vmware.com/v1alpha1:ClassClaim:psql-1
+Class Reference: 
+  Name: postgresql-unmanaged
+Parameters: 
+  storageGB: 3
+Status: 
+  Ready: True
+  Claimed Resource: 
+    Name: 7974379c-7b4d-41c3-af57-f4f1ae08c65d
+    Namespace: dev-team-1
+    Group: 
+    Version: v1
+    Kind: Secret
+```
+
+## Binding
+
+```console
+$ tanzu apps workload create emoji -n dev-team-1 \
+  -f https://raw.githubusercontent.com/tsalm-pivotal/emoji-inclusion/main/config/workload.yaml \
```

Comment:
**thought** Is this available anywhere as an accelerator or something? We may also want to lock this down to a particular tag or sha to avoid breaking changes.